cloud-formation/d2dbastion/scripts/bastion_bootstrap.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash

```

Comment:
I'm unclear what this is doing specific to a bastion server? Or is this something that should be applied to any CloudFormation-provisioned EC2 instances?